spec/support/cops/c_to_a_cop.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Test
+      class CtoA < RuboCop::Cop::Base
```

Comment:
Might be a good idea to a line to each cop mentioning it's used only for infinite loop tests.

Reply:
Added class comment to `AtoB`, `BtoC`, `CtoA` and `BtoA`